### ISSUE TITLE
Panic: Attempt to multiply with overflow

### ISSUE THREAD

**Issue by NervousNullPtr** [JASory/ENT#1]
When testing for the primality of `9223372036854775783`, a panic occurs with the following types:
-`u64`
-`i64`
-`i128`.

When testing for the primality of `922337203`, a panic occurs with the following types:
-`i32`
-`u32`
-`i64`
-`u64`
-`i128`
-`u128`.

the thread panics at the attempt to multiply with overflow at `number-theory-0.0.1/src/fjprime64.rs:1388:9`. Panic occurs when attempting to run `LibRapidRust/src/math/benchmarks::primes_benchmark`.

```powershell
9223372036854775783u128.is_prime():
thread 'benchmarks::primes_benchmark' panicked at 'attempt to multiply with overflow', /home/nervousnullptr/.cargo/registry/src/github.com-1ecc6299db9ec823/number-theory-0.0.1/src/fjprime64.rs:1388:9
stack backtrace:
   0: rust_begin_unwind
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/std/src/panicking.rs:517:5
   1: core::panicking::panic_fmt
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/core/src/panicking.rs:100:14
   2: core::panicking::panic
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/core/src/panicking.rs:50:5
   3: number_theory::fjprime64::fjprime_64
             at /home/nervousnullptr/.cargo/registry/src/github.com-1ecc6299db9ec823/number-theory-0.0.1/src/fjprime64.rs:1388:9
   4: number_theory::eightbytes::<impl number_theory::traits::NumberTheory for u64>::is_prime
             at /home/nervousnullptr/.cargo/registry/src/github.com-1ecc6299db9ec823/number-theory-0.0.1/src/eightbytes.rs:17:6
   5: number_theory::sixteenbytes::<impl number_theory::traits::NumberTheory for u128>::is_prime
             at /home/nervousnullptr/.cargo/registry/src/github.com-1ecc6299db9ec823/number-theory-0.0.1/src/sixteenbytes.rs:11:15
   6: <u128 as lib_rapid::math::primes::Primality>::is_prime
             at ./src/math/primes.rs:127:6
   7: lib_rapid::benchmarks::primes_benchmark
             at ./src/benchmarks.rs:44:7
   8: lib_rapid::benchmarks::primes_benchmark::{{closure}}
             at ./src/benchmarks.rs:35:1
   9: core::ops::function::FnOnce::call_once
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/core/src/ops/function.rs:227:5
  10: core::ops::function::FnOnce::call_once
             at /rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/core/src/ops/function.rs:227:5
```

**Comment by JASory** [comment]
Known issue. Low priority. 

In the meantime, use release mode.

**Comment by JASory** [comment]
Fixed in number-theory 0.0.2
